### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Tests pass?   | ❌

Planned work:
- [x] Show toolbar
- [ ] Rewrite the part of `DefaultController` that  needed for my next point
- [ ] Implement showing raw **(!)** result of debug process in toolbar

### CODE REVIEW

**Comment by samdark** [comment]
https://github.com/yiisoft/yii-demo/pull/63

File: src/Controllers/ToolbarController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Viewer\Controllers;
+
+use Nyholm\Psr7\Response;
```
Comment (samdark) [inline src/Controllers/ToolbarController.php]: Can we use response factory instead of relying on concrete response implementation?

Comment (xepozz) [reply src/Controllers/ToolbarController.php]: Yes, sure

**Comment by samdark** [comment]
[Yii debug](https://trello.com/c/r3CdJUaP/1-yii-debug)

**Comment by samdark** [comment]
Decided together with @xepozz to close this one.

**Pull request closed without merging**